pkg/migrations/migrations_test.go

```diff
@@ -400,6 +400,8 @@ func TestLeaseExpiration(t *testing.T) {
 	exitCalled := make(chan bool)
 	log.SetExitFunc(func(int) { exitCalled <- true })
 	defer log.SetExitFunc(os.Exit)
+	// Disable stack traces to make the test output in teamcity less deceiving.
+	defer log.DisableTracebacks()()
```

Comment:
Potentially, yeah. Although as long as `DisableTracebacks` is only used in tests, it's not going to break anything. Do you have a different suggestion?